From 832ab0c340dd8c8688058d31fd0672038e83aaff Mon Sep 17 00:00:00 2001
From: Elliott Clark <eclark@apache.org>
Date: Fri, 26 Sep 2014 18:08:27 -0700
Subject: [PATCH] HBASE-12110 Fix .arcconfig

Summary: Not many people are currently using arc but it's a nice tool for the developers who are used to it. Since it's already there let's make it work.

Test Plan: If this works then it's tested.

Differential Revision: https://reviews.facebook.net/D24075
---
 .arcconfig | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/.arcconfig b/.arcconfig
index 882b280..9cdb37d 100644
--- a/.arcconfig
+++ b/.arcconfig
@@ -2,12 +2,5 @@
   "project_id" : "hbase",
   "conduit_uri" : "https://reviews.facebook.net/",
   "copyright_holder" : "Apache Software Foundation",
-  "phutil_libraries" : {
-    "arclib" : ".arc_jira_lib"
-  },
-  "arcanist_configuration" : "ArcJIRAConfiguration",
-  "jira_project" : "HBASE",
-  "jira_api_url" : "https://issues.apache.org/jira/si/",
-  "lint_engine" : "JavaLintEngine",
   "max_line_length" : 100
 }
-- 
2.0.0

